index.js

```diff
@@ -84,10 +84,18 @@ module.exports = (input, opts) => {
 		});
 	}
 
-	tasks.add({
-		title: 'Bumping version',
-		task: () => exec('npm', ['version', input])
-	});
+	tasks.add([
+		{
+			title: 'Bumping version using Yarn',
```

Comment:
That's an excellent question - I opted for consistency with the other usage of `opts.yarn` in earlier commands to avoid bundling a refactor with a new feature.